src/NServiceBus.Core/Recoverability/Recoverability.cs

```diff
@@ -18,8 +18,11 @@ public Recoverability()
                 "Message recoverability is only relevant for endpoints receiving messages.");
             Defaults(settings =>
             {
-                settings.SetDefault(ImmedidateRetriesEnabled, true);
-                settings.SetDefault(DelayedRetriesEnabled, true);
+                settings.SetDefault(SlrNumberOfRetries, DefaultSecondLevelRetryPolicy.DefaultNumberOfRetries);
+                settings.SetDefault(SlrTimeIncrease, DefaultSecondLevelRetryPolicy.DefaultTimeIncrease);
+
+                settings.SetDefault(FlrNumberOfRetries, 5);
```

Comment:
we have to configure the defaults somewhere. I think it makes sense to do that in the `Defaults` section?